scripts/deploy-commands.js

```diff
@@ -17,16 +18,13 @@ If not, see <https://www.gnu.org/licenses/>.
 // You need to `tsc` before running this script.
 
 require("dotenv").config();
-
 const { REST } = require("@discordjs/rest");
 const { Routes } = require("discord.js");
+const CommandImportService = require("../dist/services/CommandImportService").default;
 
-const { getOrLoadAllCommands } = require("../dist/handlers/commandHandler");
-const { getApiToken, getGuildId, getClientId } = require("../dist/helpers/configHelpers");
-
-const API_TOKEN = getApiToken();
-const CLIENT_ID = getClientId();
-const GUILD_ID = getGuildId();
+const API_TOKEN = process.env.DISCORD_API_TOKEN;
+const CLIENT_ID = process.env.CLIENT_ID;
+const GUILD_ID = process.env.GUILD_ID;
```

Comment:
```suggestion
const { API_TOKEN, CLIENT_ID, GUILD_ID } = process.env;
```

Reply:
Hmm I committed this a bit too early, I think it should be `const { DISCORD_API_TOKEN: API_TOKEN, CLIENT_ID, GUILD_ID } = process.env;`. BBut it's an improvement, thanks 👍